src/state/accountCalculators.ts

```diff
@@ -91,8 +92,9 @@ export const calculateIsAccountLoading = createSelector(
  * @description calculate whether positions table should render triggers column
  */
 export const calculateShouldRenderTriggersInPositionsTable = createSelector(
-  [calculateIsAccountViewOnly],
-  (isAccountViewOnly: boolean) => !isAccountViewOnly && testFlags.configureSlTpFromPositionsTable
+  [calculateIsAccountViewOnly, getSelectedNetwork],
+  (isAccountViewOnly: boolean, selectedNetwork: DydxNetwork) =>
+    !isAccountViewOnly && ENVIRONMENT_CONFIG_MAP[selectedNetwork].featureFlags.isSlTpEnabled
```

Comment:
couldn't directly call `useEnvFeatures()` here due to a circular dep 🤔 Even when I tried directly importing the file, got build issues on vercel (but not locally). Given this is a temp flag, inclined to leave this like this for now rather than digging in further
<img width="1052" alt="Screenshot 2024-04-29 at 10 28 00 AM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/f2f265fd-66f1-45bf-85ae-456ba451061d">